content/en/docs/contribute/new-content/new-features.md

```diff
@@ -131,15 +132,15 @@ content is not received, the feature may be removed from the milestone.
 If your feature is an Alpha or Beta feature and is behind a feature gate,
 you need a feature gate file for it inside
 `content/en/docs/reference/command-line-tools-reference/feature-gates/`.
-The name of the file should be the feature gate, converted from `UpperCamelCase`
-to `kebab-case`, with `.md` as the suffix.
+The name of the file should be the name of the feature gate with `.md` as the suffix.
```

Comment:
To reviewers: this is the sole change you may want to focus on. All the other changes are about typesetting, i.e. readability and maintainability.